bridge-sdk/connectors/omni-connector/src/omni_connector.rs

```diff
@@ -421,6 +423,61 @@ impl OmniConnector {
         Ok(outcome)
     }
 
+    pub async fn bind_token_with_evm_prover(&self, tx_hash: TxHash) -> Result<CryptoHash> {
+        let eth_endpoint = self.eth_endpoint()?;
+
+        // keccak(DeployToken(address,string,string,string,uint8))
+        let event_topic =
+            H256::from_str("0x47f94ffff8bf287abb76c2b671306c7cb14769c4652a1d2fa447722e82107719")
```

Comment:
@olga24912 Why can't we compute this hash dynamically?

Reply:
It's how I started it in other connectors. We can introduce a keccak dependency and calculate it, I just haven't found time for it yet

Reply:
I can create a refactor PR for that today. Will it be a useful change or it's better to ignore it for now?

Reply:
Would be nice

Reply:
Let's remove `log_index` from bind token command and merge this

Reply:
I did that but now someone else needs to approve @frolvanya

Reply:
Done, but we have a conflict in Cargo.lock file. Can you please resolve it?